### PR TITLE
Feat: Added custom scrollbar

### LINKED ISSUE #748: [Feature Request]:  Add a customized scroll-bar in the website
### Project

Website (potterdb.com)

### Describe the feature you want to request

There could be a customized scroll bar for the website to make it look more amazing .

### (Optional) Describe a solution for this feature

With out scrollbar it looks this this
image

If a custom scroll bar is added , it would look more magical
image

### (Optional) Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### PR DESCRIPTION
**Pull request by Arbtrage**
## Summary

Added custom scrollbar in the new website.

## Related Issues
Fixes #748 

## Checklist

- [x] This pull request has a meaningful title and description.
- [x] I have read and followed the [Contributing guidelines](https://github.com/danielschuster-muc/potter-db/blob/master/CONTRIBUTING.md).
- [x] My code follows the conventions and coding style of this project.
- [x] All tests pass and I have added tests for my changes 
- [x] I have updated necessary documentation 

# Screenshots (if applicable)

image

### CODE REVIEW

File: frontend/src/styles/globals.css [hunk 1]
```diff
@@ -24,3 +24,13 @@
...
   }
 }
+/* width */
+::-webkit-scrollbar {
+  width: 8px;
+}
+ 
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: #d500f9;
```
Comment (danielschuster-muc) [inline frontend/src/styles/globals.css:34]:
```suggestion
  background: #D926D9; 
```
should use secondary color

File: frontend/src/styles/globals.css [hunk 2]
```diff
@@ -24,3 +24,13 @@
     @apply bg-background text-foreground;
   }
 }
+/* width */
+::-webkit-scrollbar {
+  width: 8px;
```
Comment (danielschuster-muc) [inline frontend/src/styles/globals.css:29]: make it a bit wider, e.g. 12 or 15 pxs

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Could you also add a scrollbar track, maybe with the primary color (#65098E)?

File: frontend/src/styles/globals.css [hunk 3]
```diff
@@ -24,3 +24,15 @@
...
+
+::-webkit-scrollbar {
+  width: 12px;
+}
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 5px #65098E; 
+  border-radius: 10px;
+}
+::-webkit-scrollbar-thumb {
+  background: #D926D9 ;
```
Comment (danielschuster-muc) [inline frontend/src/styles/globals.css:36]:
```suggestion
  background: var(--secondary);
```

File: frontend/src/styles/globals.css [hunk 4]
```diff
@@ -24,3 +24,15 @@
     @apply bg-background text-foreground;
   }
 }
+
+::-webkit-scrollbar {
+  width: 12px;
+}
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 5px #65098E;
```
Comment (danielschuster-muc) [inline frontend/src/styles/globals.css:32]:
```suggestion
  box-shadow: inset 0 0 5px var(--primary); 
```

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
I think it makes sense to use the css variables

File: frontend/src/styles/globals.css (hunk 4 above)
Comment (Arbtrage) [reply frontend/src/styles/globals.css:32]: I was using the css variables at first but for some reason the changes were not showing up on the site . Like no scrollbar was being visible .
image

Comment (danielschuster-muc) [reply frontend/src/styles/globals.css:32]: well then better use the #colors

Comment (danielschuster-muc) [reply frontend/src/styles/globals.css:32]: I think you have to wrap it with hsl, like this:
`hsl(var(--primary))`

Comment (Arbtrage) [reply frontend/src/styles/globals.css:32]: Yes , it's working now. Thanks for the suggestion.

**Review by danielschuster-muc — APPROVED** [review]
Looks good 👍

**Pull request merged**